core/controlplane/config/config.go

```diff
@@ -520,10 +523,15 @@ type Experimental struct {
 
 type Admission struct {
 	PodSecurityPolicy  PodSecurityPolicy  `yaml:"podSecurityPolicy"`
+	AlwaysPullImages   AlwaysPullImages   `yaml:"alwaysPullImages"`
 	DenyEscalatingExec DenyEscalatingExec `yaml:"denyEscalatingExec"`
 	Initializers       Initializers       `yaml:"initializers"`
 }
 
+type AlwaysPullImages struct {
+	Enabled bool `yaml:"enabled"`
```

Comment:
I suggest leaving this disabled by default. Had one issue when the docker registry could not be accessed and some health checks failed, making it impossible to restart the pods. Also, when a health check fails, it takes more to restart.

Reply:
@camilb Enabled is a true/false field and believe that struct is simply defining Enabled as a bool to be based off of the yaml value, I don't believe I'm telling it to enable it by default. If you look at the other files, and the cluster.yaml template config, I'm specifying it as false by default.

Reply:
Sorry, I was looking on the wrong field.